models/comments.models.js

```diff
@@ -0,0 +1,31 @@
+const db = require("../db/connection")
+
+exports.selectComments = (review_id) => {
+    return db
+    .query(
+        `
+        SELECT *
+        FROM reviews
+        WHERE review_id = $1;
+        `, [review_id]
+    )
+    .then(({ rows }) => {
+    if (rows.length === 0){
+        return Promise.reject({status: 404, message: "Review_id not found"})
+    } else {
+        return db
+        .query(
+            `
+            SELECT *
+            FROM comments
+            WHERE review_id = $1
+            ORDER BY created_at DESC
+            `, [review_id]
+        )
+        .then(({rows}) =>{
+            return rows
+        })
```

Comment:
you have a nested .then here, please move the .then to the end of current .then block to avoid nesting. The logic should be the same, it will just need to be moved